src/script/page/LeftSidebar/panels/Conversations/Conversations.tsx

```diff
@@ -206,9 +202,6 @@ const Conversations: React.FC<ConversationsProps> = ({
           data-uie-name="status-name"
           role="presentation"
           tabIndex={TabIndex.FOCUSABLE}
-          // personal user won't see availability status menu, on blur of the userName
```

Comment:
why did you remove this code?

// personal user won't see availability status menu, on blur of the userName
          // conversation list should get the focus
          onBlur={() => focusConversationList(true)}